core/src/main/java/org/elasticsearch/gateway/PrimaryShardAllocator.java

```diff
@@ -476,8 +476,10 @@ static NodeShardsResult buildVersionBasedNodeShardsResult(ShardRouting shard, bo
      * recovered on any node
      */
     private boolean recoverOnAnyNode(IndexMetaData metaData) {
+        // don't use the settings directly, not to trigger manny deprecation logging
```

Comment:
Nits: `settings` -> `setting` and `not to trigger manny` -> `to not trigger verbose` (it doesn't have to be exactly this, but at least fix `manny`)